### DISCUSSION TITLE
Why is the size of an image as checked with Docker commands different from the size shown in Harbor?

### DISCUSSION THREAD

**Discussion by hyeyoung-leee** [goharbor/harbor#20289] [Q&A] [answered]
Hi,

Why is the size of an image as checked with Docker commands different from the size shown in Harbor?
```
$ docker images busybox
REPOSITORY   TAG       IMAGE ID       CREATED         SIZE
busybox      latest    ba5dc23f65d4   11 months ago   4.26MB
```

<img width="829" alt="스크린샷 2024-04-17 오후 12 12 02" src="https://github.com/goharbor/harbor/assets/120620071/95fd1130-2591-4c75-b7d4-0b4fffe60a2d">

**Comment by chlins** [comment, marked as answer]
The image size displayed by using docker images locally may be after decompression, while on harbor it may be after compression.
